.orchestra/config/components/flamegraph.yml

```diff
@@ -0,0 +1,27 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+#@ load("/lib/create-component.lib.yml", "single_build_component")
+
+#@ commit = "cd9ee4c4449775a2f867acf31c84b7fe4b132ad5"
+#@ hash = "c5d180a54401074ee844009ff5fca2c003af992bc9735fc27aaf4722e21f9455"
+
+#@yaml/text-templated-strings
+---
+#@ def flamegraph():
+configure: |
+  mkdir -p "$BUILD_DIR"
+  cd "$BUILD_DIR"
+
+  fetch.sh \
+    --hash 'sha256-(@= hash @)' \
+    'https://raw.githubusercontent.com/brendangregg/FlameGraph/(@= commit @)/flamegraph.pl'
+install: |
+  cd "$BUILD_DIR"
+  install -m 755 flamegraph.pl "${DESTDIR}${ORCHESTRA_ROOT}/bin" 
```

Comment:
I've got a `flame` that gives me flame graphs:

```
#!/usr/bin/env bash

perf script -i "$1" | perl -w stackcollapse-perf.pl > out.perf-folded
perl -w flamegraph.pl out.perf-folded > perf.svg
```

Can we maybe install something like this?

Reply:
I'm not sure I follow, what is this script supposed to do?

Reply:
No, it's not related. It's just a nice to have script so that you can quickly create a flamgraph given some `perf.data` file.

Reply:
Added as `perf-flamegraph`